scripts/update-and-check-binary.sh

```diff
@@ -0,0 +1,57 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+BASE_URL="https://binaries.soliditylang.org/bin"
+REPO_ROOT="$(dirname "$0")/.."
+
+function fail() {
+	echo -e "$@" >&2
+	exit 1
+}
+
+function check_version() {
+	current_version=$(node ./dist/solc.js --version | sed -En 's/^(.*).Emscripten.*/\1/p')
+
+	# Retrieve the correspondent released version
+	short_version=$(echo "${current_version}" | sed -En 's/^([0-9.]+).*\+commit\.[0-9a-f]+.*$/\1/p')
+	release_version=$(curl -s "${BASE_URL}/list.json"  | jq ".releases | .[\"${short_version}\"]" | tr -d '"' | sed -En 's/^soljson-v(.*).js$/\1/p')
+
+	# check if current version exists as release
+	if [ "${current_version}" != "${release_version}" ]; then
```

Comment:
Yes indeed. I usually use with curly bracket because it is safer, in case for some mistake you forget to add it when the variable is surrounded by strings, like you described.

Reply:
I think it's fine to use it if you prefer that. I only skip it in that one (but very common) case for less visual noise.